util/misc.py

```diff
@@ -429,3 +433,20 @@ class can go away.
         return _new_empty_tensor(input, output_shape)
     else:
         return torchvision.ops.misc.interpolate(input, size, scale_factor, mode, align_corners)
+
+
+# Copied from Detectron2:
+# https://github.com/facebookresearch/detectron2/blob/af866c423b5897c8ab709a405c86a45b51ad4785/detectron2/structures/masks.py#L20
+def polygons_to_bitmask(polygons: List[np.ndarray], height: int, width: int) -> np.ndarray:
```

Comment:
This is already present in https://github.com/facebookresearch/detr/blob/9db8be19e76d22ee4ec770cf8ea26301151ec6a8/datasets/coco.py#L33-L47

I think we should not add this to `util.misc` , but just modify the D2 wrapper code to do the conversions we need, by calling the necessary D2 functions there.